nocrash.py

```diff
@@ -31,8 +31,6 @@
 if ChatExchangeU is None:
     ChatExchangeU = str(input("Username: ")).strip('\r\n')
 
-os.environ['CEU'] = "h"
-
 if ChatExchangeP is None:
     ChatExchangeP = str(input("Password: ")).strip('\r\n')
```

Comment:
Could you change this so that it uses `getpass` like [this](https://github.com/iBug/SmokeDetector/blob/a8598353fb78b5db98065e182d6e16d825b523a1/ws.py#L72) line? That way the password won't show up in plain text.